crates/flipperzero/src/furi/string.rs

```diff
@@ -35,8 +35,8 @@ const WHITESPACE: &[char] = &[
 ///
 /// This is similar to Rust's [`CString`] in that it represents an owned, C-compatible,
 /// nul-terminated string with no nul bytes in the middle. It also has additional methods
-/// to provide the flexibility of Rust's [`String`]. It is used in various APIs of the
-/// Flipper Zero SDK.
+/// to provide the flexibility of Rust's [`String`].
+/// It is used in various APIs of the Flipper Zero SDK.
```

Comment:
This change is no longer necessary:
```suggestion
/// to provide the flexibility of Rust's [`String`]. It is used in various APIs of the
/// Flipper Zero SDK.
```

Reply:
I think that it is more readable now (when looking at docs in sources) since the new sentence starts from the new line while still maintaining approximately same column width.